revscoring/datasources/meta/vectorizers.py

```diff
@@ -62,3 +64,29 @@ def load_kv(filename=None, path=None, limit=None):
         else:
             raise TypeError(
                 "load_kv() requires either 'filename' or 'path' to be set.")
+
+
+class FastText(Datasource):
+    """
+    Generates vectors using FastText for a list of items generated
+    by another datasource.
+
+    :Parameters:
+        words_datasource : :class:`revscoring.Datasource`
+            A datasource that returns a list of words.
+        vectorize_words : `function`
+            a function that takes a list of words and converts it to a list
+            of vectors of those words
+        name : `str`
+            A name for the `revscoring.FeatureVector`
+    """  # noqa
+
+    def __init__(self, words_datasource, vectorize_words, name=None):
+        name = self._format_name(name, [words_datasource, vectorize_words])
+
+        super().__init__(name, vectorize_words, depends_on=[words_datasource])
+
+    def vectorize_words(vector_model, words):
+        model = fasttext.load_model(vector_model)
```

Comment:
This is a big operation.  It'll run every time we want to process a set of words.  I it takes 3-4 seconds on my computer.

Reply:
Yep it's a big operation indeed. As you advised earlier, I've worked on a new update that re-uses the word2vec datasource since the fasttext `.vec` files are "word2vec format".